### PR TITLE
Document lookup tables

### PR DESCRIPTION
**Pull request by bernd**

### CODE REVIEW

File: pages/lookuptables.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+**Example 2:** Output for the geo-ip data adapter including a single value
+and a multi value.
+
+.. image:: /images/lookuptables/multi-value-example.png
+
+Setup
+-----
+
+The lookup tables can be configured on the "System/Lookup Tables" page.
```
Comment (jalogisch) [inline pages/lookuptables.rst:80]: in other parts of the Docs we use `System > Lookup Tables` (Code Markup) that should be identical in all parts

**Review by jalogisch — COMMENTED** [review]
I would change that before commit - but please comment.

File: pages/lookuptables.rst (hunk 1 above)
Comment (bernd) [reply pages/lookuptables.rst:80]: We are mostly using the ` / ` notation:

```
$ git grep -e 'System >' -e 'System>' | wc -l
3
$ git grep -e 'System /' -e 'System/' | wc -l
29
```

In some places we also use ` -> `. Might be nice to unify that at one point.

**Review by jalogisch — APPROVED** [review]
here we go

File: pages/lookuptables.rst (hunk 1 above)
Comment (jalogisch) [reply pages/lookuptables.rst:80]: i'll cover that

**Pull request merged**